grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -154,9 +177,9 @@ private Json stopTask(Request request, Response response) {
             @ApiImplicitParam(name = "createdBy", value = "String representing the user scheduling this task", required = true, dataType = "string", paramType = "query"),
             @ApiImplicitParam(name = "priority", value = "String representing priority of a task. Must be one of {high, low}. The default is \"low\"", required = false, dataType = "string", paramType = "query", access = "low", allowableValues = "high,low"),
             @ApiImplicitParam(name = "runAt", value = "Time to run at as milliseconds since the UNIX epoch", required = true, dataType = "long", paramType = "query"),
-            @ApiImplicitParam(name = "interval",value = "If set the task will be marked as recurring and the value will be the time in milliseconds between repeated executions of this task. Value should be as Long.",
+            @ApiImplicitParam(name = "interval", value = "If set the task will be marked as recurring and the value will be the time in milliseconds between repeated executions of this task. Value should be as Long.",
                     dataType = "long", paramType = "query"),
-            @ApiImplicitParam(name = "configuration", value = "JSON Object that will be given to the task as configuration.", dataType = "String", paramType = "body")
+            @ApiImplicitParam(name = REST.Request.CONFIGURATION_PARAM, value = "JSON Object that will be given to the task as configuration.", dataType = "String", paramType = "body")
```

Comment:
This is pedantic- if we use the variable for one of these comments, we should for all of them. I like the idea! Keeps us consistent.